### PR TITLE
Enable to pass html options to URL field

### PR DESCRIPTION
**Pull request by shouichi**
Close https://github.com/thoughtbot/administrate/issues/2134.

### CODE REVIEW

File: lib/administrate/field/url.rb [hunk 1]
```diff
@@ -11,6 +11,10 @@ def truncate
         data.to_s[0...truncation_length]
       end
 
+      def html_options
+        @options[:html_options] || {}
+      end
```
Comment (shouichi) [inline lib/administrate/field/url.rb:16]: I have two questions.

1. Is the method name OK?
2. Should we automatically add `rel="noopener"` for security?

Comment (nickcharlton) [reply lib/administrate/field/url.rb:16]: 1. I think the method name is okay — it's the same as what Rails calls it.
2. No, I think that'd be surprising for the end user.

**Review by nickcharlton — COMMENTED** [review]
This looks good!

One thought though: Do you think we could move the options up to the `Base` class so that it applies to all fields?

**Comment by shouichi** [comment]
> One thought though: Do you think we could move the options up to the Base class so that it applies to all fields?

Sounds cool! I searched the codebase (`git grep content_tag`) and found that only `Fields::Url` uses `content_tag`. The templates of some fields are somewhat non-trivial. It might not be obvious where the `html_options` applies to. I believe we can gradually add `html_options` to certain fields, then introduce it to the base class when we find it useful (and we can do that in a backward-compatible way I guess). What do you think?

By the way, I realized that I also needed to change `_index.html.erb` template too. I'll fix it tomorrow.

Thanks!

**Review by nickcharlton — APPROVED** [review]
Thanks for updating!

Yes, I think we can leave it as just the URL field for now and expand it over time.

**Pull request merged**